Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.12-slim
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .

```

Comment:
Using `COPY . .` can lead to larger image sizes and potentially include unnecessary files. Ensure that your `.dockerignore` file is properly configured to exclude files that are not needed in the image.